shared/constants/network.ts

```diff
@@ -529,7 +635,41 @@ export const BUYABLE_CHAINS_MAP: {
   [CHAIN_IDS.MAINNET]: {
     nativeCurrency: CURRENCY_SYMBOLS.ETH,
     network: BUYABLE_CHAIN_ETHEREUM_NETWORK_NAME,
-    transakCurrencies: [CURRENCY_SYMBOLS.ETH, 'USDT', 'USDC', 'DAI'],
+    transakCurrencies: [
+      CURRENCY_SYMBOLS.ETH,
+      '1INCH',
+      'AAVE',
+      'AGEUR',
+      'BUSD',
+      'CHAIN',
+      'CLV',
+      'COMP',
+      'CTSI',
+      'DAI',
+      'DAO',
+      'ENJ',
+      'EURT',
+      'GTH',
+      'HEX',
+      'LINK',
+      'MANA',
+      'MASK',
+      'MINDS',
+      'MKR',
+      'PLA',
+      'POLS',
+      'SAND',
+      'STETH',
+      'SUSHI',
+      'SWAP',
+      'TXL',
+      'UNI',
+      'USDC',
+      'USDT',
+      'VRA',
+      'WBTC',
+      'YLD',
```

Comment:
use the CURRENCY_SYMBOLS constant here so that the types aren't widened to strings.